styles/Home.module.scss

```diff
@@ -5,7 +5,7 @@
 
   .cards__wrapper {
     display: grid;
-    grid-template-columns: repeat(auto-fit, minmax(350px, 1fr));
+    grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
```

Comment:
please do this change only in mobile

Reply:
Should I add `@media only screen and (max-width: 600px)`?

Reply:
yep